core/unit_test/incremental/Test05_ParallelReduce_RangePolicy.hpp

```diff
@@ -139,16 +141,18 @@ struct TestReduction {
 
     // parallel_reduce call with range policy over num_elements number of
     // iterations
-    Kokkos::parallel_reduce("Reduction", range_policy(0, num_elements),
+    Kokkos::parallel_reduce("Reduction", range_policy(0, m_num_elements),
                             ReduceFunctor(deviceData), sum);
 
     check_correctness_and_cleanup();
   }
 };
 
 TEST(TEST_CATEGORY, IncrTest_05_reduction) {
-  TestReduction<TEST_EXECSPACE> test;
-  test.sum_reduction();
+  for (unsigned int i = 1; i < 100; ++i) {
```

Comment:
I had some issues that the implementation would only work for certain values so I wanted to test some more here.
Note that zero-size reductions don't yet work.